src/jobManagerClient.ts

```diff
@@ -10,208 +10,186 @@ import {
-          err,
-          Object.getOwnPropertyNames(err)
-        )}`
-      );
+      this.logger.error({ err, url: createTaskUrl, targetService: this.targetService, payload }, `failed to enqueueTask for jobId=${jobId}`);
       throw err;
     }
   }
 
   public async updateTask<T>(jobId: string, taskId: string, payload: IUpdateTaskBody<T>): Promise<void> {
+    const updateTaskUrl = `/jobs/${jobId}/tasks/${taskId}`;
     try {
-      this.logger.info(`[JobManagerClient][updateTask] jobId=${jobId}, taskId=${taskId}, payload=${JSON.stringify(payload)}`);
-      const updateTaskUrl = `/jobs/${jobId}/tasks/${taskId}`;
+      this.logger.info(
+        { url: updateTaskUrl, targetService: this.targetService, jobId, taskId, payload },
+        `updateTask for jobId=${jobId}, taskId=${taskId}`
+      );
       await this.put(updateTaskUrl, payload);
     } catch (err) {
       this.logger.error(
-        `[JobManagerClient][updateTask] jobId=${jobId}, taskId=${taskId}, payload=${JSON.stringify(payload)} failed error=${JSON.stringify(
-          err,
-          Object.getOwnPropertyNames(err)
-        )}`
+        { err, url: updateTaskUrl, targetService: this.targetService, jobId, taskId, payload },
+        `failed to updateTask for jobId=${jobId}, taskId=${taskId}`
       );
       throw err;
     }
   }
 
   public async findTasks<T>(criteria: IFindTaskRequest<T>): Promise<ITaskResponse<T>[] | undefined> {
-    this.logger.debug(`[JobManagerClient][findTasks] finding task with data: ${JSON.stringify(criteria)}`);
     const findTaskUrl = '/tasks/find';
+    this.logger.debug({ url: findTaskUrl, targetService: this.targetService, criteria }, `findTasks`);
     try {
       const res = await this.post<ITaskResponse<T>[]>(findTaskUrl, criteria);
       return res;
     } catch (err) {
       if (err instanceof NotFoundError) {
-        this.logger.debug(`[JobManagerClient][findTasks] no tasks match specified criteria`);
+        this.logger.debug({ url: findTaskUrl, targetService: this.targetService }, `findTasks response - no tasks match specified criteria`);
         return undefined;
       } else {
-        this.logger.error(`[JobManagerClient][findTasks] filed to search tasks: ${JSON.stringify(err, Object.getOwnPropertyNames(err))}`);
+        this.logger.error({ err, url: findTaskUrl, targetService: this.targetService, criteria }, `failed to findTasks`);
         throw err;
       }
     }
   }
 
   public async createJob<T, P>(payload: ICreateJobBody<T, P>): Promise<ICreateJobResponse> {
+    const createJobUrl = `/jobs`;
     try {
-      this.logger.info(`[JobManagerClient][createJob] payload=${JSON.stringify(payload)}`);
-      const createJobUrl = `/jobs`;
+      this.logger.info({ url: createJobUrl, targetService: this.targetService, payload }, `createJob`);
       const res = await this.post<ICreateJobResponse>(createJobUrl, payload);
```

Comment:
mabye debug\ info log of new job creation with is and status

Reply:
fixed